e2e/Microsoft.Azure.Devices.E2ETests.NetStandard/Microsoft.Azure.Devices.E2ETests.NetStandard.csproj

```diff
@@ -12,6 +12,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
+    <CommonSrc>..\..\common\src</CommonSrc>
```

Comment:
where is this used?

Reply:
Microsoft.Azure.Devices.E2ETests.NetStandard.csproj